WalletWasabi.Fluent/Styles/Themes/Base.xaml

```diff
@@ -11,10 +11,11 @@
     <Color x:Key="SystemAccentColorDark3">#FF2E8041</Color>
     <Color x:Key="SystemAccentColor">#FF72BD81</Color>
     <SolidColorBrush x:Key="TextControlBorderBrushFocused" Color="{StaticResource SystemAccentColor}"/>
-    <SolidColorBrush x:Key="TextControlErrorBorderBrush">#D85050</SolidColorBrush>
+    <SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush">#D85050</SolidColorBrush>
     <SolidColorBrush x:Key="ActionButtonForegroundColor">#FFFFFF</SolidColorBrush>
```

Comment:
This should be an alias to one of the 32 base resources...

Alias can be defined like https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Accents/BaseDark.xaml#L152

Reply:
Done.

Reply:
hmmm that wasnt quite what I meant!

ok we need to talk about this in meeting on Wednesday... for now its good.